includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -627,7 +628,7 @@ public static function filter_row_actions( $actions, $post ) {
 		$actions[ self::VALIDATE_ACTION ] = sprintf(
 			'<a href="%s">%s</a>',
 			esc_url( self::get_recheck_url( $post ) ),
-			esc_html__( 'Re-check', 'amp' )
+			esc_html__( 'Recheck', 'amp' )
```

Comment:
Removed hyphen from 'Re-check' for consistency.

Reply:
Good, as it seems the most commonly-accepted spelling is without the hyphen.